app/controllers/SearchController.java

```diff
@@ -337,11 +337,11 @@ protected String determineHistogramResolution(final SearchResult searchResult) {
             Collections.sort(usedIndices, new Comparator<IndexRangeSummary>() {
                 @Override
                 public int compare(IndexRangeSummary o1, IndexRangeSummary o2) {
-                    return o1.start().compareTo(o2.start());
+                    return o1.begin().compareTo(o2.begin());
```

Comment:
@kroepke Are you sure this is correct? The "start()" method maps to the "end()" method of the new index range class.